src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -129,10 +128,18 @@ def compress(
                 if name.endswith("zero_point") and self._skip_zp(name, names_to_scheme):
                     continue
 
+                if name.endswith("weight_scale") and self._skip_scale():
+                    continue
+
                 compressed_dict[name] = value.to(compression_device)
 
         return compressed_dict
 
+    def _skip_scale(self):
```

Comment:
why are we skipping scales just for nvfp4? it has scales right?

Reply:
Any qparam that is compressed is currently handled by compress_weight because we dont have a good way to handle qparam compression (see zp for the packed compressor). As a result, we skip it when iterating through the params to prevent the dense qparam from over-writing it.